contracts/SwapKiwi.sol

```diff
@@ -200,43 +244,45 @@ contract SwapKiwi is Ownable, IERC721Receiver {
   }
 
   /**
-    * @dev Returns NFTs from SwapKiwi to swap initator.
-    *      Callable only if second user hasn't yet added NFTs.
-    *
-    * @param swapId ID of the swap that the swap participants want to cancel
-    */
-  function cancelSwap(uint256 swapId) external {
+  * @dev Returns NFTs from SwapKiwi to swap initator.
+  *      Callable only if second user hasn't yet added NFTs.
+  *
+  * @param swapId ID of the swap that the swap participants want to cancel
+  */
+  function cancelSwap(uint64 swapId) external {
     require(
       _swaps[swapId].initiator == msg.sender || _swaps[swapId].secondUser == msg.sender,
       "SwapKiwi: Can't cancel swap, must be swap participant"
     );
-      // return initiator NFTs
-      safeMultipleTransfersFrom(
-        address(this),
-        _swaps[swapId].initiator,
-        _swaps[swapId].initiatorNftAddresses,
-        _swaps[swapId].initiatorNftIds
-      );
+    // return initiator NFTs
+    safeMultipleTransfersFrom(
+      address(this),
+      _swaps[swapId].initiator,
+      _swaps[swapId].initiatorNftAddresses,
+      _swaps[swapId].initiatorNftIds,
+      _swaps[swapId].initiatorNftAmounts
+    );
 
     if(_swaps[swapId].secondUserNftAddresses.length != 0) {
```

Comment:
```suggestion
    if(_swaps[swapId].secondUserNftAddresses.length > 0) {
```

Reply:
If we have this condition here, then lets have this condition everywhere where we call safeMultipleTransfersFrom.